tests/templates/spark-shuffle-job.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: "sparkoperator.k8s.io/v1beta2"
+kind: SparkApplication
+metadata:
+  name: {{ .Name }}
+  namespace: {{ .Namespace }}
+spec:
+  type: Scala
+  mode: cluster
+  image: {{ .Image }}
+  imagePullPolicy: Always
+  mainClass: ShuffleApp
+  mainApplicationFile: "https://infinity-artifacts.s3.amazonaws.com/scale-tests/dcos-spark-scala-tests-assembly-2.4.0-20190325.jar"
+  arguments:
+    - "4"
+    - {{ index .Params "group_count" }}
+    - "100"
+    - "4"
+    - "1500"
```

Comment:
do we want to parameterize the rest of the arguments so we can vary them from the test code only?

Reply:
ok, i've parameterized all arguments for this app